Upstream: No, Alpine specific
Reason: We don't want to log into $PWD by default but into /var/log/mautrix-telegram.log

diff --git a/example-config.yaml b/example-config.yaml
index 2c2040a..927e8b6 100644
--- a/example-config.yaml
+++ b/example-config.yaml
@@ -399,7 +399,7 @@ logging:
         file:
             class: logging.handlers.RotatingFileHandler
             formatter: normal
-            filename: ./mautrix-telegram.log
+            filename: /var/log/mautrix-telegram.log
             maxBytes: 10485760
             backupCount: 10
         console:
